docs/EXTENDED-ARCHIVES.md

```diff
@@ -101,9 +96,10 @@ described above ensures the maximum UX perceived efficiency.
 
 ### Graphics
 
-We chose [`svg`][svg] over other image formats since clients can display it on
-any kind of screen resolution without requiring multi-layer files, or multiple
-files workarounds.
+Any graphic included in the archive can be either an [`svg`][svg] file, or a
+[`png`][png] file with the `<filename>.png` and `<filename>@2x.png` convention.
```

Comment:
I think we should allow:
* no filenames
* `filename.svg` only
* `filename.png` only
* `filename.png` and `filename@2x.png`
* `filename.png` and `filename@2x.png` and `filename.svg` (include both PNG and SVG in case some clients don't support displaying SVGs?)

The more flexibility we allow, the easier it'll be for publishers to start using the extra features.

I guess that also suggests that maybe graphics should be referenced in the manifest by the extension-less part of the filename, so the client is then free to choose the 'best' extension from those available?